### PR TITLE
Update release notes based on r13.0.0rc2 review

### PR DESCRIPTION
**Pull request by git-harry**
The release notes for r13.0.0rc2 were reviewed in preparation for the
release of r13.0.0. Of the changes to be agreed based on that review the
following are relevant to newer versions of the notes:

- a test note in the repo is removed
- a note about L3HA has been expanded to explain why it's being disabled
- the prelude sections have been removed because they weren't adding
  value
- references to VXLAN interfaces have been replaced with VXLAN networks
  to be more accurate
- a note regarding overlay networks has been reworded to better explain
  the change to which it refers
- security hardening note has been adjusted to keep to control interface
  consistent for all deployment paths
- the removal of LVMISCSIDriver by Cinder is noted

Connected https://github.com/rcbops/u-suk-dev/issues/398

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-be0a447781/docs/private-cloud/release-notes/). :bow:

**Comment by git-harry** [comment]
recheck_swift

**Comment by git-harry** [comment]
recheck_ceph

**Comment by evrardjp** [comment]
Same comment as #1403 . Except that we are fine IMO.

File: releasenotes/notes/turn-off-l3ha-18274d19d54a6315.yaml [hunk 1]
```diff
@@ -12,9 +12,8 @@ upgrade:
     For more information, see:
     `bug 1149 <https://github.com/rcbops/rpc-openstack/issues/1149>`_
     or the known issues section in the v13.0 RPCO documentation.
-fixes:
-  - Fixed `bug 1149 <https://github.com/rcbops/rpc-openstack/issues/1149>`_
 other:
-  - Neutron L3HA as a default has been disabled.
+  - Neutron L3HA as a default has been disabled. This has been done
+    to due to the number of bugs that still exist in that feature.
```
Comment (mancdaz) [inline releasenotes/notes/turn-off-l3ha-18274d19d54a6315.yaml]: s/to due to/due to'

**Comment by mancdaz** [comment]
One inline comment

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-477fb60183/docs/private-cloud/release-notes/).

**Comment by sigmavirus24** [comment]
👍 This is similar to the other review with the exception that it doesn't remove upgrade notes.

Thanks @git-harry

**Comment by evrardjp** [comment]
👍

**Pull request merged**